lib/src/hmssdk.dart

```diff
@@ -410,4 +410,32 @@ class HMSSDK {
   void removeLogListener({required HMSLogListener hmsLogListener}) {
     PlatformService.removeLogsListener(hmsLogListener);
   }
+
+  Future<void> startHlsStreaming(String meetingUrl,
```

Comment:
move start & stop HLS functions to just below stopRecording API i.e. to line 348